### PR TITLE
CAL-1450 : Add the user avatar to calender participants suggester

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: calendar-webapp/src/main/webapp/vue-app/calServices.js [hunk 1]
```diff
@@ -237,7 +237,16 @@ export function findParticipants(filter, limit) {
     limit = DEFAULT_LIMIT;
   }
   return fetch(`${calConstants.CAL_SERVER_API}participants?name=${filter}&limit=${limit}`, {headers: calConstants.HEADER_NO_CACHE})
-    .then(resp =>  resp.json()).then(json => json.data);
+    .then(resp =>  resp.json()).then(json => {
+      const glabalUsers = [];
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/webapp/vue-app/calServices.js]: I guess it is a typo, it should be globalUsers. I would even rename it to `participants`.

File: calendar-webapp/src/main/webapp/vue-app/calServices.js [hunk 2]
```diff
@@ -237,7 +237,16 @@ export function findParticipants(filter, limit) {
...
+      for (const usr of json.data )
+      {
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/webapp/vue-app/calServices.js]: Brackets must be at the end of the line.

File: calendar-webapp/src/main/webapp/vue-app/calServices.js [hunk 3]
```diff
@@ -237,7 +237,16 @@ export function findParticipants(filter, limit) {
...
+        const av = {avatar: `/rest/v1/social/users/${usr.id}/avatar`};
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/webapp/vue-app/calServices.js]: Shouldn't the /participants endpoint return the avatar URL instead ? The advantages is that the service would really be restful, the client does not need to guess the avatar URL (as we do now). So we can even change it in the future. WDYT ?

File: calendar-webapp/src/main/webapp/vue-app/calServices.js (hunk 1 above)
Comment (SaraBoutej) [reply calendar-webapp/src/main/webapp/vue-app/calServices.js]: Done !

File: calendar-webapp/src/main/webapp/vue-app/calServices.js (hunk 2 above)
Comment (SaraBoutej) [reply calendar-webapp/src/main/webapp/vue-app/calServices.js]: Done

File: calendar-webapp/src/main/webapp/vue-app/calServices.js (hunk 3 above)
Comment (SaraBoutej) [reply calendar-webapp/src/main/webapp/vue-app/calServices.js]: The endpoint /participants (/rest/v1/calendar/participants) didn't contain avatar URL instead, it contains only (email, name, id, href) that's why i added the avatar manually to the data!
If it's necessary i will try to add the avatar URL in the rest service

Comment (thomasdelhomenie) [reply calendar-webapp/src/main/webapp/vue-app/calServices.js]: Yes, let's try that.

Comment (SaraBoutej) [reply calendar-webapp/src/main/webapp/vue-app/calServices.js]: Done !

**Comment by SaraBoutej** [comment]
> `LinkProvider.buildAvatarURL(OrganizationIdentityProvider.NAME, username)`)

I'll try that !

**Comment by SaraBoutej** [comment]
> (you can use `LinkProvider.buildAvatarURL(OrganizationIdentityProvider.NAME, username)`). WDYT ?

Done !

**Pull request merged**